VAMobile/src/screens/HealthScreen/SecureMessaging/ReplyMessage/ReplyMessage.test.tsx

```diff
@@ -28,6 +28,24 @@ jest.mock('store/slices', () => {
         payload: '',
       }
     }),
+    resetSaveDraftComplete: jest.fn(() => {
```

Comment:
Had to update this so the unit tests worked once I deleted the claims and appeals slice. no idea why it was working before deletion, but whatever lol